src/main/java/uk/nhs/tis/trainee/notifications/model/NotificationType.java

```diff
@@ -45,6 +45,7 @@ public enum NotificationType {
   GMC_REJECTED_LO("gmc-rejected-lo"),
   GMC_REJECTED_TRAINEE("gmc-rejected-trainee"),
   GMC_UPDATED("gmc-updated"),
+  LTFT_ADMIN_UNSUBMITTED("ltft-admin-unsubmitted"),
```

Comment:
I have used the convention LTFT[_source if not default source]_[STATE][_destination if not trainee]